packages/graphback-datasync/src/providers/DataSyncConflictProvider.ts

```diff
@@ -3,7 +3,7 @@ import { DataSyncModelConflictConfig, DataSyncFieldNames, ConflictMetadata, Conf
 import { MongoDeltaSource } from '../deltaSource';
 import { DataSyncMongoDBDataProvider } from './DatasyncMongoDBDataProvider';
 
-export const MAX_RETRIES = 3;
+export const MAX_RETRIES = process.env.CONFLICT_RESOLUTION_MAX_RETRIES || 3;
```

Comment:
Since `process.env` values must be a string, you will need to make a helper to do a conversion.

You can use [`parseInt`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt) to convert from string to int, and is `isNan` set it to 3 as the default.